packages/core/src/external/fhir/bundle/convert-to-transaction-bundle.ts

```diff
@@ -0,0 +1,51 @@
+import { Bundle, BundleEntry } from "@medplum/fhirtypes";
```

Comment:
Leaving a comment that this code is unchanged and just moved is helpful

Reply:
I literally made the comment and forgot to press "finish review"... Sorry!